kubernetes/cmsweb/scripts/deploy-keys-secrets.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# helper script to deploy robot, proxy and token secrets in all namespaces.
+
+if [ $# -ne 2 ]; then
+    echo "Usage: deploy-key-secrets.sh <namespace>  <path_to_keys>"
```

Comment:
please add clear description what this script should do, i.e. put line 2 over here.